server/routes/api/teas.js

```diff
@@ -33,65 +34,63 @@ router.post("/new-tea", (req, res) => {
 
     newTea
       .save()
-      .then(tea => res.json(tea))
+      .then(tea => res.json(teaNormalizer(tea)))
       .catch(err => console.log(err));
-
   });
 });
 
-
 router.put("/update-tea", (req, res) => {
-
-  Tea.findOneAndUpdate({id: req.body.id}, req.body, {new: true}, function(err, doc){
+  Tea.findOneAndUpdate({ id: req.body.id }, req.body, { new: true }, function(
+    err,
+    tea
+  ) {
     if (err) return res.send(500, { error: err });
-    return res.json(doc);
+    return res.json(teaNormalizer(tea));
   });
+});
 
-})
-
-router.delete("/delete-tea", (req, res) => {
+router.delete("/delete-tea/:id", (req, res) => {
   // The "tea" in this callback function represents the document that was found.
   // It allows you to pass a reference back to the client in case they need a reference for some reason.
-  Tea.findOneAndDelete({id: req.body.id},{}, (err, todo) => {
+  Tea.findOneAndDelete({ id: req.params.id }, {}, (err, todo) => {
     // As always, handle any potential errors:
     if (err) return res.status(500).send(err);
     // We'll create a simple object to send back with a message and the id of the document that was removed
     // You can really do this however you want, though.
     const response = {
-        message: "Tea successfully deleted",
-        id: req.body.id
+      message: "Tea successfully deleted",
+      id: req.body.id
     };
     return res.status(200).send(response);
   });
 });
 
 // get the tea with that id (accessed at GET http://localhost:8080/api/teas/:tea_id)
-router.get('/tea', (req, res) => {
-  Tea.findOne({id: req.body.id},(err, tea) => {
-    if (err){
+router.get("/tea", (req, res) => {
+  Tea.findOne({ id: req.body.id }, (err, tea) => {
+    if (err) {
       res.send(err);
     }
-    if(tea){
+    if (tea) {
       res.json(tea);
```

Comment:
that api point hasn't been set up yet. i have a note to implement it later. it's meant to grab a single tea, like for the Details view. right now nothing points to this. i can add the normalizer, but nothing is going to trigger it.

Reply:
There's no consumer of this endpoint, but the endpoint itself looks like it's ready to be used.  It'll return a key based on the id it's passed.  It should return a normalized tea to follow the pattern you've established elsewhere in your API (especially since you've already done all the work and just need to pass `tea` into the normalizer).